demo/aws/aws-tomcat/configure-tomcat.tf

```diff
@@ -0,0 +1,106 @@
+resource "aws_security_group" "ubuntu-tomcat" {
+  name        = "ubuntu-tomcat-security-group"
```

Comment:
Replace `ubuntu-...` prefix with `test-..` or `sample-...` for clarity, in case if someone forgets to destroy these resources, so we can trace them.

Reply:
done